bundles/org.openhab.binding.folderwatcher/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="folderwatcher"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="ftpfolder">
+		<label>FTP Folder</label>
+		<description>FTP folder to be watched</description>
+
+		<channels>
+			<channel id="newftpfile" typeId="newfile-channel" />
```

Comment:
Wouldn't it be more intuitive from the user perspective if this channel and the one below had the same name like `newfile`?

Reply:
I was thinking about that, but how would I specify it in FolderWatcherBindingConstants.java then? Currently I have:
    public static final String CHANNEL_FILENAME = "newftpfile";
    public static final String CHANNEL_LOCALFILENAME = "newlocalfile";

Reply:
@cpmeister could you also comment on this?

Reply:
There is no reason they couldn't have the same channel id. Since these channels would belong to different things there would be no conflict.

Reply:
A channel's UID is a combination of the ThingUID and a channel id. The user interacts with channels through their UID not their id.